.gitignore

```diff
@@ -2,6 +2,7 @@ bin/
 dev/
 testbin/
 *.test
+alerttestdata
```

Comment:
you should use the `dev/` directory for this

Reply:
Yep done